downloader.py

```diff
@@ -82,7 +82,9 @@ def download_comments_new_api(youtube_id, sleep=1):
                          for ncd in search_dict(response, 'nextContinuationData')] + continuations
 
         for comment in search_dict(response, 'commentRenderer'):
-            yield {'cid': comment['commentId'],
+            cid = comment['commentId']
+            yield {'cid': cid,
+                   'parent_id': cid.split('.')[0] if '.' in cid else None,
```

Comment:
I don't really like storing fields that can be derived from other fields. This is best done during post-processing.